vdirel.el

```diff
@@ -91,7 +91,7 @@ Return nil if PROPERTY is not in CONTACT."
 (defun vdirel-contact-emails (contact)
   "Return a list of CONTACT's email addresses."
   (vdirel--contact-matching-properties
-   (lambda (property) (string-match "^EMAIL" property))
+   (lambda (property) (string-match "^\\(item.?\\.\\)?EMAIL" property))
```

Comment:
optional: I believe the code would get clearer by using `rx`. What do you think?

Reply:
Have updated the request with a version using `rx` for the regex.

Reply:
thank you